content/guides/component-testing/testing-angular.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Testing Angular Components
+---
+
+Now that the component is mounted, the next step is to start selecting and
+interacting with parts of the component. This is the **Act** step in "Arrange,
+Act, Assert".
+
+Once we are done acting on the component, we can then verify the expected state
+of the component is what we think it should be. This is the **Assert** step.
+
+## Selecting the StepperComponent
+
+By default, the Stepper component's counter is initialized to "0". It also has
+an `@Input` property that can specify an initial count from a parent component.
+
+Let's test that mounting the component (Arrange) in its default state has a
+count of "0" (Assert).
+
+Then, we will test that setting the initial count also works.
+
+In your spec file, add the following inside the existing `describe` block:
+
+<code-group>
+<code-block label="stepper.component.cy.ts" active>
+
+```ts
+// Set up some constants for the selectors
```

Comment:
same wrapper vs template here. If the reasoning was to include wrapper examples because the Vue guide shows both component and JSX examples, the reasoning there was that even though we felt using JSX was better, its not natural in the Vue ecosystem, show we always showed both ways. In Angular, using the template syntax is natural and preferred, so a wrapper example only makes sense when there are real benefits to doing so.

Not going to point this out in every example from here on out but I think I would feel the same about all the rest of them.